Backend/dhall-configs/dev/rider-app.dhall

```diff
@@ -199,6 +200,7 @@ let jobInfoMapx =
         , mapValue = False
         }
       , { mapKey = RiderJobType.OtherJobTypes, mapValue = False }
+      , { mapKey = AllocatorJobType.MetroIncentivePayout, mapValue = True }
```

Comment:
please add this in triggerScheduler api to trigger this job

Reply:
We will run a query to add jobData to Db directly